verifiers/internal/gha/provenance.go

```diff
@@ -294,17 +294,42 @@ func isValidDelegatorBuilderID(prov iface.Provenance) error {
 	if err != nil {
 		return err
 	}
+
+	sourceURI, err := prov.SourceURI()
+	if err != nil {
+		return err
+	}
+
+	uri, _, err := utils.ParseGitURIAndRef(sourceURI)
```

Comment:
You aren't using the value `uri` before overwriting it at L323. Is that intentional?

Reply:
yes. Let me rename the second uri to normalizedUri

Reply:
done:
normalizedE2eRepoUri and normalizedUri